static/css/s2.css

```diff
@@ -12983,6 +12983,151 @@ span.ref-link-color-3 {color: blue}
     width: 67%;
   }
 }
+
+.product {
+  max-width: 600px;
+}
+
+#productsPageContent {
+  margin-top: 225px;
+  margin-inline-start: 68px;
+}
+
+.product hr {
+  border: 1px solid var(--light-grey);
+}
+
+.productsInner {
+  display: flex;
+  align-items: right;
+}
+
+.productsInner img {
+  max-width: 100%; 
+  max-height: 116px;
+  height: auto;
+  display: block;
+  margin-inline-end: 3%;
+}
+
+.productsInner .productsDesc {
+  font: 'Roboto', sans-serif; 
+  color: #666666;
+  font-size: 16px;
+}
+
+.productsDesc p {
+  margin-top: 0;
+}
+
+.productsHeader {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10%;
+}
+
+.productsTitleAndLabel {
+  flex: auto;
+}
+
+.productsHeader .productsTitle {
+  font: 'Roboto', sans-serif; 
+  color: #666666;
+  font-size: 22px;
+  font-weight: 500px;
+}
+
+.productsHeader .productsTypeLabel {
+  font: 'Roboto', sans-serif; 
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 18px;
+  background-color: #EDEDEC; 
+  border-radius: 6px; 
+  padding: 0.01px 5px;
+  color: #666666; 
+  margin-inline-start: 3%;
+}
+
+.productsHeader .cta {
+  display: flex;
+  align-items: center;
+}
+
+.productsHeader .cta .productsCTA {
+  color: #4B71B7; 
+  border-width: 10px;
+  border-color: #0cd200;
+  margin-inline-end: 10px;
+}
+
+.productsHeader .cta .productsCTAIcon {
+  width: 13.33px;
+  height: 16px;
+  top: 646px;
+  left: 610px;
+  margin: 0 5px;
+  filter: brightness(0) saturate(100%) invert(52%) sepia(17%) saturate(6763%) hue-rotate(200deg) brightness(78%) contrast(77%);;
```

Comment:
what are we doing here?

Reply:
Re-coloring the icon SVGs to match the spec. I found it much easier to contain in one line like this versus other approaches. What would you suggest?

Reply:
I hear that. I'll try to get the other approach working again.

Reply:
Got the other approach working, @edamboritz curious what you think